app/models/invoice.rb

```diff
@@ -465,10 +464,10 @@ def charge!(automatic = true)
       # REFACTOR: Move the logic to InvoiceMessenger
       if charging_retries_count < MAX_CHARGE_RETRIES
         if unpaid?
-          logger.info("Retrying #{self.id}, unpaid")
+          logger.info("Invoice #{id} is unpaid, will be retried")
```

Comment:
```suggestion
          logger.info("Invoice #{id} is still unpaid, will be retried")
```

Reply:
Changed it to 
`"Invoice #{id} (buyer #{buyer_account_id}) remains unpaid after #{charging_retries_count} attempts, will be retried"` just to reflect it's status and the fact that it does not change